Makefile

```diff
@@ -189,8 +189,17 @@ helm-install-druid-operator: ## helm upgrade/install
 	helm upgrade --install \
 	--namespace ${NAMESPACE_DRUID_OPERATOR} \
 	--create-namespace \
+	zookeeper bitnami/zookeeper
+	helm upgrade --install \
+	--namespace ${NAMESPACE_DRUID_OPERATOR} \
+	--create-namespace \
+	--set auth.username=druid \
+	--set auth.password=druid \
+	--set auth.database=druid \
+	mysql bitnami/mysql
+	helm upgrade --install \
+	--namespace ${NAMESPACE_DRUID_OPERATOR} \
 	${NAMESPACE_DRUID_OPERATOR} chart/ \
-	--set image.repository=${IMG_KIND} \
```

Comment:
Why was this removed?

Reply:
i've adjusted it back for now. Switched to developing the app locally with telepresence v2, so i have easy access to services and the k8s api.